### PR TITLE
Counters

### PR DESCRIPTION
**Pull request by MeikeWeiss**
Issue #175 and #190 

I'm not sure if it's always the smartest way and if the names are well chosen.

### CODE REVIEW

File: doc/ExampleApplications.xml [hunk 1]
```diff
@@ -104,9 +104,9 @@ gap&gt; EulerCharacteristic(newSurf);
         be represented as a triangulation of the sphere. We might want to
         know how many vertices there are of different degrees. This
         can be found out by using <K>VertexCounter</K>
```
Comment (TomGoertzen) [inline doc/ExampleApplications.xml:106]: Shouldn't this also say CounterOfVertices here? There is another occurrence of VertexCounter in the manual beside this. I am not sure if this is intended :)

File: doc/ExampleImplementations.xml [hunk 2]
```diff
@@ -190,7 +190,7 @@ true
     </Section>
 
     <Section Label="Example_VertexCounter"><Heading>VertexCounter</Heading>
-        The method <K>VertexCounter</K> (<Ref Subsect="VertexCounter"/>) 
+        The method <K>VertexCounter</K> (<Ref Subsect="CounterOfVertices"/>)
```
Comment (TomGoertzen) [inline doc/ExampleImplementations.xml:193]: Is VertexCounter an alias for CounterOfVertices?

File: gap/PolygonalComplexes/properties.gd [hunk 3]
```diff
@@ -275,7 +275,13 @@ DeclareAttribute( "BlockType",IsSimplicialSurface);
 #! These two definitions are distinguished by calling them 
 #! <K>EdgeDegreesOfVertices</K> and <K>FaceDegreesOfVertices</K>.
 #! 
-#TODO mention vertexCounter, edgeCounter
+#! A large part of this chapter are the counters. There are six different counter objects:
+#! vertex counter(<Ref Subsect="CounterOfVertices"/>), edge counter(<Ref Subsect="CounterOfEdges"/>),
+#! face counter(<Ref Subsect="CounterOfFaces"/>), butterfly counter(<Ref Subsect="CounterOfButterflies"/>),
+#! umbrella counter(<Ref Subsect="CounterOfUmbrellas"/>) and three face counter(<Ref Subsect="CounterOfThreeFaces"/>).
+#! The counters save information about the degrees. There are different methods to get these information
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/properties.gd:282]: Maybe: For a given surface, the counters save information about the vertex, face or edge degrees.

File: doc/ExampleImplementations.xml (hunk 2 above)
Comment (TomGoertzen) [reply doc/ExampleImplementations.xml:193]: I mean, that the function VertexCounter also exists and does the same as CounterOfVertices. Is this the case?

**Review by TomGoertzen — DISMISSED** [review]
Nice :)

**Review by TomGoertzen — APPROVED** [review]

**Review by ReymondAkpanya — APPROVED** [review]

**Pull request merged**